sms-quickstart.rb

```diff
@@ -0,0 +1,10 @@
+require 'rubygems'
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)